tests/core/test_scheduler.py

```diff
@@ -27,14 +30,18 @@ def test_scheduler_add_seq_group():
     # Add seq group to scheduler.
     num_seq_group = 4
     for i in range(num_seq_group):
-        _, seq_group = create_dummy_prompt(str(i), block_size)
+        _, seq_group = create_dummy_prompt(str(i),
+                                           block_size,
+                                           block_size=block_size)
```

Comment:
Good fix. I noticed this problem for a long while but just too lazy to fix it...